.changeset/grumpy-feet-listen.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/ui5-application-writer': minor
```

Comment:
That is a mistake, correct? Or is there a reason for a minor version increase of the app-writer?

Reply:
Correct, changeset needed to be updated @tobiasqueck